pkg/extensions/extensions.go

```diff
@@ -55,9 +55,13 @@ func EnableExtensions(extension *ExtensionConfig, log log.Logger, rootDir string
 }
 
 // SetupRoutes ...
-func SetupRoutes(router *mux.Router, storeController storage.StoreController, log log.Logger) {
+func SetupRoutes(extension *ExtensionConfig, router *mux.Router, storeController storage.StoreController,
+	log log.Logger) {
 	log.Info().Msg("setting up extensions routes")
-	resConfig := search.GetResolverConfig(log, storeController)
-	router.PathPrefix("/query").Methods("GET", "POST").
-		Handler(gqlHandler.NewDefaultServer(search.NewExecutableSchema(resConfig)))
+
+	if extension.Search != nil && extension.Search.Enable {
```

Comment:
I think we are missing one case here, if user did not set enable to true but provided cve config, we will not setup /query routes but we will still download cve database. 

pkg/extensions/extensions.go 
``` 
// Enable extensions if extension config is provided		if c.Config != nil &amp;&amp; c.Config.Extensions != nil {			ext.EnableExtensions(c.Config.Extensions, c.Log, c.Config.Storage.RootDirectory)		}
```

and  https://github.com/andaaron/zot/blob/cli1/pkg/extensions/extensions.go#L35

Reply:
Updated: https://github.com/anuvu/zot/pull/201/files#diff-86b8c7ae55e63b332b1b61b1e50093ca23babbdf0ad78ff1f74bcbec501cc188R36